internal/service/advancedcluster/resource_advanced_cluster_migration_test.go

```diff
@@ -177,7 +127,7 @@ func TestMigAdvancedCluster_geoShardedTransitionFromOldToNewSchema(t *testing.T)
 		CheckDestroy: acc.CheckDestroyCluster,
 		Steps: []resource.TestStep{
 			{
-				ExternalProviders: mig.ExternalProviders(),
+				ExternalProviders: acc.ExternalProviders("1.17.5"), // last version that did not support new sharding schema or attributes
```

Comment:
```suggestion
				ExternalProviders: acc.ExternalProviders("1.17.6"), // last version that did not support new sharding schema or attributes
```

Reply:
nit: should we just make "1.17.6" a constant? since this particular version would be quite relevant for a while, naming it would help understand the significance